apollo-ios/Sources/Apollo/MultipartResponseDeferParser.swift

```diff
@@ -64,12 +64,14 @@ struct MultipartResponseDeferParser: MultipartResponseSpecificationParser {
         }
 
       case let .json(object):
-        if let hasNext = object.hasNext {
-          preconditionFailure("This will be done in #3147")
-        }
+        if let _ = object.incremental {
```

Comment:
We don't need to check `.incremental` and `.object`. We do the same serialization either way. So let's just do it.

Reply:
Done in https://github.com/apollographql/apollo-ios-dev/pull/217/commits/92c094355eda77f85969f28d92132b501586672a.